modules/ROOT/pages/tutorial-containers.adoc

```diff
@@ -193,22 +193,26 @@ CONTAINER ID  IMAGE                       COMMAND               CREATED        S
 And we can set a key/value pair in etcd. For now let’s set the key `fedora` to the value `fun`:
 
 ----
-[core@tutorial ~]$ curl -L -X PUT http://127.0.0.1:2379/v2/keys/fedora -d value="fun"
-{"action":"set","node":{"key":"/fedora","value":"fun","modifiedIndex":4,"createdIndex":4}}
-[core@tutorial ~]$ curl -L http://127.0.0.1:2379/v2/keys/ 2>/dev/null | jq .
+[core@tutorial ~]$ curl -L http://127.0.0.1:2379/v3/kv/put -X POST -d '{"key": "ZmVkb3Jh", "value": "ZnVu"}'
```

Comment:
Can we still have the key = `fedora` and the value = `fun` ?

Reply:
I don´t think so. As far as I understook in the documentation it needs to be a base64 string.

Reply:
ahh. I see: 

```
[dustymabe@media ~]$ echo -n 'fedora' | base64 
ZmVkb3Jh
```

Well that's no fun :)